platform/core/src/defaults/hotkeyBindings.js

```diff
@@ -88,6 +88,20 @@ const bindings = [
   //   keys: ['pagedown'],
   //   isEditable: true,
   // },
+  {
+    commandName: 'nextStage',
+    context: 'DEFAULT',
+    label: 'Next Stage',
+    keys: ['ctrl+end'],
```

Comment:
I am not crazy about using these as the default hot keys. home and end are usually for the first and last respectively of some operation. How about Ctrl+a and Ctrl+s? Or just 'a' and 's'? Or ',' and '.'? I was thinking 's' for stage. Anyway, home and end I think should be reserved for something else. Just my two cents. Maybe it does not matter.

Reply:
Ok how about Ctrl+Left Arrow and Ctrl+Right Arrow? Are those already used?